src/ducks/notifications/BalanceLower/index.js

```diff
@@ -147,14 +148,19 @@ class BalanceLower extends NotificationView {
   }
 
   getTitle(templateData) {
-    const { accounts } = templateData
+    const { accounts, matchingRules } = templateData
     const onlyOne = accounts.length === 1
     const firstAccount = accounts[0]
 
     const titleKey = `Notifications.if_balance_lower.notification.${
-      onlyOne ? 'one' : 'several'
+      onlyOne
+        ? 'one'
+        : matchingRules.length === 1
+        ? 'several'
+        : 'several-multi-rule'
```

Comment:
That's a good occasion to make it pass the grep test!

Reply:
Done,good idea !